eclair-core/eclair-cli

```diff
@@ -44,6 +44,7 @@ and COMMAND is one of the available commands:
     - allchannels
     - allupdates
     - channelstats
+    - channelbalances
```

Comment:
I think we should be consistent, if you put it in the `Channel` section instead of the `Payment` section, then the code for this API should be in `handlers/Channel.scala` (you currently put it next to `usablebalances` in `handlers/Payment.scala`).

Or we keep the code in `handlers/Payment.scala` and this put it in the `Payment` section here, whatever you prefer.

Reply:
I think it's better to move it to `handlers/Channel.scala` since it is not specifically intended to be used for payments like `usablebalances` is.

Reply:
fixed in [a897eb0](https://github.com/ACINQ/eclair/pull/2196/commits/a897eb0f0a3b64e02434f17c28c91596b3a3969c)